security/pkg/pki/util/verify_cert.go

```diff
@@ -59,6 +58,7 @@ func VerifyCertificate(privPem []byte, certChainPem []byte, rootCertPem []byte,
 		return err
 	}
 
+	host := expectedFields.Host
 	san := host
```

Comment:
Not introduced by your PR but I think you can remove `san` variable and replace all references with `host`.

Reply:
@incfly Thanks for your review! BTW, the CI results really confuse me.

Reply:
@incfly We can't remove `san` since there are places using `host` (the value of  `san` may change). https://github.com/istio/istio/blob/63150a1500572779887f67f7d49b9ba23584c09c/security/pkg/pki/util/verify_cert.go#L65-L66)

Reply:
Good catch.